components/automate-ui/src/app/page-components/insight-attributes-dropdown/insight-attributes-dropdown.model.ts

```diff
@@ -0,0 +1,50 @@
+export interface FilterOption {
+  name: FilterName;
+  id: string;
+}
+
+export enum FilterName {
+  Platform = 'Platform',
+  PlatformVersion = 'Platform Version',
```

Comment:
Hi, @SEAjamieD , I chatted with @lancewf about this, and it seems that we have combined the `Platform` and `PlatformVersion` into one single attribute, so the value becomes something like `ubuntu 18.04`. I suggest we remove the option of `Platform` from the UI and keep only the `Platform Version` for it.